Test suite checking

I just ran the test suite using the following commands

```bash
git clone https://github.com/adtzlr/felupe.git
cd felupe
conda create --name felupe-review
conda activate felupe review
conda install pip pytest
pip install --editable .
pytest
```

and get the following error.

```bash
==================================================================================================== ERRORS ====================================================================================================
_________________________________________________________________________________ ERROR collecting tests/test_constitution.py __________________________________________________________________________________
ImportError while importing test module 'C:\Users\nguarinz\Workspace\felupe\tests\test_constitution.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\anaconda3\envs\felupe-review\Lib\importlib\__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests\test_constitution.py:27: in <module>
    import matplotlib.pyplot as plt
E   ModuleNotFoundError: No module named 'matplotlib'
_______________________________________________________________________________ ERROR collecting tests/test_constitution_jax.py ________________________________________________________________________________
ImportError while importing test module 'C:\Users\nguarinz\Workspace\felupe\tests\test_constitution_jax.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\anaconda3\envs\felupe-review\Lib\importlib\__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests\test_constitution_jax.py:27: in <module>
    import jax.numpy as jnp
E   ModuleNotFoundError: No module named 'jax'
=========================================================================================== short test summary info ============================================================================================
ERROR tests/test_constitution.py
ERROR tests/test_constitution_jax.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================================== 2 errors in 11.34s ==============================================================================================
```

I just created an environment from scratch and used

    pip install --editable ".[all]"

instead in the block of commands shown above and obtained the following

```bash
=============================================================================================== warnings summary ===============================================================================================
tests/test_job.py: 11 warnings
  C:\Users\nguarinz\Workspace\felupe\src\felupe\mechanics\_solidbody_gravity.py:98: DeprecationWarning: felupe.SolidBodyGravity is deprecated, use felupe.SolidBodyForce instead.
    self.__init__(self.field, gravity, self.results.density)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================================================================================== short test summary info ============================================================================================
FAILED tests/test_constitution.py::test_umat_hyperelastic_statevars - _tkinter.TclError: Can't find a usable init.tcl in the following directories:
FAILED tests/test_constitution.py::test_optimize - _tkinter.TclError: Can't find a usable tk.tcl in the following directories:
============================================================================ 2 failed, 168 passed, 11 warnings in 63.76s (0:01:03) ============================================================================
```

From the whole message block: "This probably means that tk wasn't installed properly."

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -91,6 +91,12 @@ or clone the repository and install the package in editable mode.
    cd felupe
    pip install --editable .
 
+Optional dependencies may also be installed by replacing the last line.
+
+.. code-block:: shell
+
+   pip install --editable ".[all]"
+
 Extension Packages
 ------------------
 

--- a/tests/test_constitution.py
+++ b/tests/test_constitution.py
@@ -24,6 +24,10 @@ You should have received a copy of the GNU General Public License
 along with Felupe.  If not, see <http://www.gnu.org/licenses/>.
 
 """
+import matplotlib
+
+matplotlib.use("Agg")
+
 import matplotlib.pyplot as plt
 import numpy as np
 import pytest

